lib/Repository.js

```diff
@@ -368,41 +371,6 @@ class Repository extends Requestable {
    }
 
    /**
-    * Update commit status
-    * @see https://developer.github.com/v3/repos/statuses/
-    * @param {string} commitSHA - the SHA of the commit that should be updated
-    * @param {object} options - Commit status parameters
-    * @param {string} options.state - The state of the status. Can be one of: pending, success, error, or failure.
-    * @param {string} [options.target_url] - The target URL to associate with this status.
-    * @param {string} [options.description] - A short description of the status.
-    * @param {string} [options.context] - A string label to differentiate this status among CI systems.
-    * @param {Requestable.callback} cb - will receive the updated commit back
-    * @return {Promise} - the promise for the http request
-    */
-   updateStatus(commitSHA, options, cb) {
```

Comment:
@nmoya i think this happened when rebasing cuz i worked on it on `master` then moved it to the other branch. i'll fix and resubmit. thanks!